README.md

```diff
@@ -1,7 +1,6 @@
 # Homebrew (Legacy)
 This repository has been deprecated and split into two repositories:
-- [Homebrew/brew](https://github.com/Homebrew/brew): the Homebrew
-  package manager without any formulae/packages (i.e. without the former contents of `Library/Formula`)
-- [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core): the formulae/packages (i.e. the former contents of `Library/Formula`) for the Homebrew package manager
+- for **formulae/packages** -> [Homebrew/**homebrew-core**](https://github.com/Homebrew/homebrew-core)
```

Comment:
I'd like to still have the `the former contents of Library/Formula` in some form. I also dislike the ascii `->`; could you just use a `:`?